vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -205,7 +205,7 @@ $global:map = @{
     );
     "c:\akse-cache\win-vnet-cni\" = @(
         "https://acs-mirror.azureedge.net/azure-cni/v1.4.22/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.22.zip",
-        "https://acs-mirror.azureedge.net/azure-cni/v1.4.35/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.35.zip",
+        "https://acs-mirror.azureedge.net/azure-cni/v1.4.41/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.41.zip",
```

Comment:
Please do not delete v1.4.35 since it is being used. We always keep N-1 or N-2 in the cache.